src/components/navbar/Navbar.jsx

```diff
@@ -45,7 +68,114 @@ const Navbar = () => {
         ))}
       </ul>
     </nav>
-  );
+  );}
+  else
+  {
+    return(
+      <div style={{position:"sticky", top:0, zIndex:100,alignItems:"center", display:"flex",flexDirection:"column"}}>
```

Comment:
Do not use inline styling in JSX, as it is very error prone. Use TailwindCSS instead for utility based CSS system, you will also have intellisense if you are using VS Code with proper extensions.